cmd/staking/root.go

```diff
@@ -121,13 +121,13 @@ func (s *staker) run(cmd *cobra.Command, args []string) error {
 			return staking.DirectiveDelegate, staking.Delegate{
 				dAddr,
 				dAddr2,
-				big.NewInt(1000),
+				big.NewInt(0).Mul(big.NewInt(denominations.One), big.NewInt(10)),
```

Comment:
What does this achieve?

Reply:
Nothing, just testing change. to be able to delegate more tokens.